### ISSUE TITLE
Improve `yarn rw test` command

### ISSUE THREAD

**Issue by thedavidprice** [redwoodjs/redwood#1150]
The Test command needs some work:
- [ ] default should be run (no watch) (see #1415) 
- [x] add watch (maybe with alias)
- [x] update descriptions
- [x] confirm there's an appropriate command option for CI
- [x] fix bug: error when path contains spaces #1279 
- [x] update docs

**Comment by peterp** [comment]
My suggestion would be the alias `yarn rw test` to `yarn jest`, so that all of the flags that are available in jest are also available in Redwood.

**Comment by jeliasson** [comment]
> 
> 
> My suggestion would be the alias `yarn rw test` to `yarn jest`, so that all of the flags that are available in jest are also available in Redwood.

I agree with @peterp. Also, `yarn rw test api` also throws an error with `URL` not being picked up in the `DS` model while `yarn jest` does not. Might be because I'm slightly behind in the the Redwood versions tho.

**Comment by thedavidprice** [comment]
@peterp ah, that's a fantastic idea if we could pull it off. Worst case we allow a string option to pass any config similar to what we did for [rw dev](https://redwoodjs.com/docs/cli-commands#dev).

@jeliasson more likely that we're still fixing/improving some things. I updated this project board with the latest status for testing; you'll notice a few things are still busted: https://github.com/redwoodjs/redwood/projects/4
